src/pages/_meta.json

```diff
@@ -2,5 +2,6 @@
   "index": "Welcome",
   "sylvia": "Sylvia",
   "cw-multi-test": "MultiTest",
+  "core": "CosmWasm Core",
```

Comment:
We probably want to reorder these at some point, since core is probably the prerequisite for all the other ones

Reply:
Yeah, definitely. Well, we can pull it up before merging